lib/msal-common/src/authority/Authority.ts

```diff
@@ -768,5 +783,29 @@ export class Authority {
 
         return metadata;
     }
-}
 
+    /**
+     * Transform CIAM_AUTHORIY as per the below rules:
+     * If no path segments found and it is a CIAM authority, then transform it
+     * hostname ends with .ciamlogin.com (a constant of-course) and any Path => do not transform
+     * hostname ends with .ciamlogin.com (a constant of-course) and noPath  => transform
```

Comment:
nit to make this a little less verbose:

```suggestion
     * If no path segments found and it is a CIAM authority (hostname ends with .ciamlogin.com), then transform it
```

Reply:
Sorry, was meaning to fix this, pasted verbatim for a discussion as a place holder :). Committed the suggestion. Doc changes and samples are in progress for end users.